pkg/kv/kvserver/mvcc_gc_queue.go

```diff
@@ -64,9 +69,10 @@ const (
 
 	// Thresholds used to decide whether to queue for MVCC GC based on keys and
 	// intents.
-	mvccGCKeyScoreThreshold          = 2
-	mvccGCIntentScoreThreshold       = 1
-	mvccGCDropRangeKeyScoreThreshold = 1
+	mvccGCKeyScoreThreshold           = 1
```

Comment:
I tried to simulate that just to see if there are some unexpected effects that we are missing, but there aren't any increased GC frequency with existing or new thresholds. What I think would happen is that once protected timestamp is moved, all ranges become eligible for GC and they have accumulated extra 24h of data so that we scan more data in each range and GC runs are all synchronized. The other issue is that score at this point reached value of 4 and randomization has no effect. So we will be grinding through ranges. I think having setting for a threshold won't help us here, but we can try to randomize better maybe.

Reply:
Experiment shows that: with PTS blocking GC we have a GC run at the point of removing PTS:
![threshold-1-remove-protected-timestamp-high-rewrite](https://user-images.githubusercontent.com/5124786/204113086-3fb92140-36ff-44b4-9c03-e9e8e990be8c.png)
And if we compare GC rate after PTS is removed, it matches exactly the steady state so there's no additional GC activity. The scan and request execution would be expensive due to extra garbage involved:
![threshold-1-steady-write-high-rewrite](https://user-images.githubusercontent.com/5124786/204113128-69423e73-0366-49f7-90d5-ebaf96176129.png)
We even have a benefit of improved collection efficiency at the first run.